CMakeLists.txt

```diff
@@ -0,0 +1,30 @@
+cmake_minimum_required(VERSION 3.16)
+project(deepbacksub CXX)
+
+find_package(OpenCV REQUIRED COMPONENTS core imgproc video videoio highgui)
+
+# This assumes that tensorflow got checked out as submodule.
+# True, if `git clone` had the additional option `--recursive`.
+add_subdirectory(tensorflow/tensorflow/lite 
+  "${CMAKE_CURRENT_BINARY_DIR}/tensorflow-lite" EXCLUDE_FROM_ALL)
```

Comment:
Why are you setting the `EXCLUDE_FROM_ALL` property here? Any reasonable makefile generator should build the build instructions in a way such that tensorflow won't be rebuilt unless you change anything in it - in which case it should be rebuilt by definition.

Reply:
This is actually from the official tensorflow-lite example; my best guess is: there are dependencies, which are not needed for each build. Or even dependencies, which could inject install rules.
In the end, this will allow users to simply issue `make`  instead of `make deepseg` and be done with it.

Reply:
k. Just wanted to make sure there's some reasoning behind this. :)